src/render/draw_symbol.js

```diff
@@ -192,6 +194,15 @@ function setSymbolDrawState(program, painter, layer, tileZoom, isText, isSDF, ro
     } else if (sizeData.isFeatureConstant && sizeData.isZoomConstant) {
         gl.uniform1f(program.u_size, sizeData.layoutSize);
     }
+    gl.uniform1f(program.u_camera_to_center_distance, tr.cameraToCenterDistance);
+    if (layer.layout['symbol-placement'] === 'line' &&
+        layer.layout['text-rotation-alignment'] === 'map' &&
+        layer.layout['text-pitch-alignment'] === 'viewport' &&
+        layer.layout['text-field']) {
+        gl.uniform1f(program.u_max_camera_distance, 1.5);
+    } else {
+        gl.uniform1f(program.u_max_camera_distance, 10);
```

Comment:
This is a ratio, so technically unit-less. But yeah, it's a ratio of "camera to label distance" over "camera to center distance".

The "1.5" value came from experimentation -- basically just looking for a value that would clip out really ugly labels but leave most in place. "10" is effectively infinite (well, at least with our limit of 60 degrees of pitch).

Passing in the uniform as a float is left over from an earlier iteration of the PR where this value was controlled per-property in the style. Now that it's hardwired, it probably makes sense to push the value into the shader and just make the uniform into a boolean (something like `u_is_pitched_line_label`).